addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/handler/BigAssFanHandler.java

```diff
@@ -483,6 +461,17 @@ private boolean isLightPresent() {
         return false;
     }
 
+    private static final StringType LIGHT_COLOR = new StringType("COLOR");
```

Comment:
I seem to have missed this the other time the normal order (element of least surprise) is that a class first contains all constants, then all fields, then constructor(s) and then the normal methods om some sensible order.

Reply:
Ok